client/app/components/about-modal/about-modal.html

```diff
@@ -0,0 +1,6 @@
+<a href="#"  ng-click="vm.open()" translate>
+  About
```

Comment:
Should be translated per the other way (see other comments), if nothing else to keep a common style.